assets/js/blocks/cart-checkout/checkout/inner-blocks/checkout-terms-block/edit.tsx

```diff
@@ -66,41 +68,86 @@ export const Edit = ( {
 				) : (
 					<RichText
 						tagName="span"
-						value={ text || termsConsentDefaultText }
+						value={ currentText }
 						onChange={ ( value ) =>
 							setAttributes( { text: value } )
 						}
 					/>
 				) }
 			</div>
-			{ ! currentText.includes( '<a ' ) && (
+			{ /* Show this notice if a terms page or a privacy page is not setup. */ }
+			{ ( ! TERMS_URL || ! PRIVACY_URL ) && (
 				<Notice
 					className="wc-block-checkout__terms_notice"
 					status="warning"
 					isDismissible={ false }
-					actions={
-						termsConsentDefaultText !== text
-							? [
-									{
-										label: __(
-											'Restore default text',
-											'woo-gutenberg-products-block'
-										),
-										onClick: () =>
-											setAttributes( { text: '' } ),
-									},
-							  ]
-							: []
-					}
+					actions={ [
+						! TERMS_URL && {
+							label: __(
+								'Setup a Terms and Conditions page',
+								'woo-gutenberg-products-block'
+							),
+							onClick: () =>
+								window.open(
+									`${ ADMIN_URL }admin.php?page=wc-settings&tab=advanced`,
+									'_blank'
+								),
```

Comment:
Using window.open because I couldn't pass `rel` to the button. I need to open the link in a new page.

Reply:
Probably not blocking, but there is a weird issue that if you click on those links with the middle button of the mouse, the warning disappears:

https://user-images.githubusercontent.com/3616980/135092488-e84e1bf2-8556-4392-84b5-fd54dd2a2388.mp4

Reply:
Hmm, that might be something specific to Linux. Clicking the middle button acts like pasting a text, I noticed that whatever I have in the clipboard appears there when clicking with the middle button.

I don't think that should block release, but at some point it might be good to investigate if we can replace these `button`'s with `a`, which I guess will help preventing this kind of issues.